wai-aria/scripts/aria-utils.js

```diff
@@ -146,5 +146,36 @@ const AriaUtils = {
   },
 
 
+  /*
+  assert_not_equals() wrapper function to verify an element's name computation
+  ignores aria-label (computedLabel != aria-label value).
+
+  Ex: <button aria-label="" data-testname="button with empty aria-label does not use aria-label as name">
+        class="ex">
+        buttonText
+        </button>
+
+      AriaUtils.verifyAriaLabelIsIgnored(".ex")
+
+  See Web Platform Test's test harness API documentation for available assert functions:
+  https://web-platform-tests.org/writing-tests/testharness-api.html#assert-functions
+
+  */
+  verifyAriaLabelIsIgnored: function(selector) {
+    const els = document.querySelectorAll(selector);
+    if (!els.length) {
+      throw `Selector passed in verifyAriaLabelIsIgnored("${selector}") should match at least one element.`;
+    }
+    for (const el of els) {
+      let testName = el.getAttribute("data-testname");
+      promise_test(async t => {
+        const elAriaLabel = el.getAttribute("aria-label");
+        let computedLabel = await test_driver.get_computed_label(el);
+        computedLabel = computedLabel.trim()
```

Comment:
This `trim()` is going to invalidate many of your whitespace tests above.

Reply:
Right! I took your whitespace updates for `verifyLabelsBySelector()` and re-factored it, adding a new genericized `verifyAttributeIsIgnoredforComputedLabel()` function. Please see: beeeb539ef03c1a346527083a93da1c668cc05d6.